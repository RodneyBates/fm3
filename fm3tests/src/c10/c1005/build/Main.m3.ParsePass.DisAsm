   361 16_00000000000000a2                   162 ItkEOF()
   359 16_00000000000000a3                   163 ItkLeftEnd()
   357 16_0000000000000167                   359 ItkExportIdListLt(L1(16_1),[1,7])
   352 16_000000000000016e                   366 ItkExportIdListElem(Id1"Main",[1,7])
   347 16_000000000000016a                   362 ItkExportIdListRt(L1(16_1),[1,7])
   342 16_00000000000000e6                   230 ItkBlockLt(L1(16_1),[3,2])
   337 16_00000000000001bf                   447 ItkBlockDeclListLt(L1(16_1),[3,2])
   332 16_00000000000000da                   218 ItkProcWBodyLt(L2(16_2))
   329 16_00000000000000c2                   194 ItkProcSignatureLt([4,4])
   325 16_000000000000019f                   415 ItkFormalsListLt(L6(16_6),[4,4])
   320 16_0000000000000149                   329 ItkVALUEFormalLt([4,6])
   316 16_00000000000001a7                   423 ItkVALUEFormalIdListLt(L2(16_2),[4,6])
   311 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id3"F1",[4,6])
   306 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id4"F2",[4,11])
   301 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L2(16_2),[4,6])
   296 16_000000000000014d                   333 ItkVALUEFormalType([4,14])
   292 16_00000000000000b4                   180 ItkReservedId(Rid"INTEGER",[4,16])
   286 16_0000000000000150                   336 ItkVALUEFormalVal([5,4])
   282 16_00000000000000ad                   173 ItkFormalExprAbsent([5,4])
   278 16_000000000000014c                   332 ItkVALUEFormalRt([4,6])
   274 16_00000000000001a3                   419 ItkFormalsListSep(L2(16_2),[5,4])
   269 16_0000000000000149                   329 ItkVALUEFormalLt([5,12])
   265 16_00000000000001a7                   423 ItkVALUEFormalIdListLt(L1(16_1),[5,12])
   260 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id5"F3",[5,12])
   255 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L1(16_1),[5,12])
   250 16_000000000000014d                   333 ItkVALUEFormalType([5,15])
   246 16_00000000000000b4                   180 ItkReservedId(Rid"CHAR",[5,17])
   240 16_0000000000000150                   336 ItkVALUEFormalVal([6,4])
   236 16_00000000000000ad                   173 ItkFormalExprAbsent([6,4])
   232 16_000000000000014c                   332 ItkVALUEFormalRt([5,6])
   228 16_00000000000001a3                   419 ItkFormalsListSep(L3(16_3),[6,4])
   223 16_000000000000015d                   349 ItkROFormalLt([6,15])
   219 16_00000000000001b7                   439 ItkROFormalIdListLt(L1(16_1),[6,15])
   214 16_00000000000001be                   446 ItkROFormalIdListElem(Id6"F4",[6,15])
   209 16_00000000000001ba                   442 ItkROFormalIdListRt(L1(16_1),[6,15])
   204 16_0000000000000161                   353 ItkROFormalType([6,18])
   200 16_00000000000000b4                   180 ItkReservedId(Rid"BOOLEAN",[6,20])
   194 16_0000000000000164                   356 ItkROFormalVal([7,4])
   190 16_00000000000000ad                   173 ItkFormalExprAbsent([7,4])
   186 16_0000000000000160                   352 ItkROFormalRt([6,6])
   182 16_00000000000001a3                   419 ItkFormalsListSep(L4(16_4),[7,4])
   177 16_0000000000000153                   339 ItkVARFormalLt([7,10])
   173 16_00000000000001af                   431 ItkVARFormalIdListLt(L1(16_1),[7,10])
   168 16_00000000000001b6                   438 ItkVARFormalIdListElem(Id7"F5",[7,10])
   163 16_00000000000001b2                   434 ItkVARFormalIdListRt(L1(16_1),[7,10])
   158 16_0000000000000157                   343 ItkVARFormalType([7,13])
   154 16_00000000000000b4                   180 ItkReservedId(Rid"WIDECHAR",[7,15])
   148 16_000000000000015a                   346 ItkVARFormalVal([8,4])
   144 16_00000000000000ad                   173 ItkFormalExprAbsent([8,4])
   140 16_0000000000000156                   342 ItkVARFormalRt([7,6])
   136 16_00000000000001a3                   419 ItkFormalsListSep(L5(16_5),[8,4])
   131 16_0000000000000149                   329 ItkVALUEFormalLt([8,6])
   127 16_00000000000001a7                   423 ItkVALUEFormalIdListLt(L1(16_1),[8,6])
   122 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id8"F6",[8,6])
   117 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L1(16_1),[8,6])
   112 16_000000000000014d                   333 ItkVALUEFormalType([9,4])
   108 16_00000000000000ac                   172 ItkFormalTypeAbsent([9,4])
   104 16_0000000000000150                   336 ItkVALUEFormalVal([9,4])
   100 16_00000000000000ad                   173 ItkFormalExprAbsent([9,4])
    96 16_000000000000014c                   332 ItkVALUEFormalRt([8,6])
    92 16_00000000000001a2                   418 ItkFormalsListRt(L6(16_6),[4,4])
    87 16_00000000000000b0                   176 ItkResultTypeAbsent([10,1])
    83 16_00000000000000ae                   174 ItkRaisesSetAbsent([10,1])
    79 16_00000000000000c5                   197 ItkProcSignatureRt([4,4])
    75 16_00000000000000de                   222 ItkProcBodyLt([10,1])
    71 16_00000000000000e6                   230 ItkBlockLt(L4(16_4),[10,3])
    66 16_00000000000001c2                   450 ItkBlockDeclListRt(L0(16_0),[10,3])
    61 16_00000000000001bf                   447 ItkBlockDeclListLt(L0(16_0),[10,3])
    56 16_00000000000000ea                   234 ItkBlockBEGIN(L4(16_4),[10,3])
    51 16_00000000000001ca                   458 ItkStmtListRt(L0(16_0),[11,3])
    46 16_00000000000001c7                   455 ItkStmtListLt(L0(16_0),[11,3])
    41 16_00000000000000e9                   233 ItkBlockRt(L4(16_4),[10,3])
    36 16_00000000000000e1                   225 ItkProcBodyRt([10,1])
    32 16_00000000000000dd                   221 ItkProcWBodyRt(L2(16_2))
    29 16_00000000000001c2                   450 ItkBlockDeclListRt(L1(16_1),[3,2])
    24 16_00000000000000ea                   234 ItkBlockBEGIN(L1(16_1),[13,2])
    19 16_00000000000001ca                   458 ItkStmtListRt(L0(16_0),[14,2])
    14 16_00000000000001c7                   455 ItkStmtListLt(L0(16_0),[14,2])
     9 16_00000000000000e9                   233 ItkBlockRt(L1(16_1),[3,2])
     4 16_00000000000000a4                   164 ItkRightEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
