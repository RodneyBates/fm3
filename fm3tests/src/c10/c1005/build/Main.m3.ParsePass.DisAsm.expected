   345 16_00000000000000a2                   162 ItkEOF()
   343 16_00000000000000a3                   163 ItkLeftEnd()
   341 16_000000000000016c                   364 ItkExportIdListLt(L1(16_1),[1,7])
   336 16_0000000000000173                   371 ItkExportIdListElem(Id1"Main",[1,7])
   331 16_000000000000016f                   367 ItkExportIdListRt(L1(16_1),[1,7])
   326 16_00000000000000ea                   234 ItkBlockLt(L1(16_1),[3,2])
   321 16_00000000000001c4                   452 ItkBlockDeclListLt(L1(16_1),[3,2])
   316 16_00000000000000de                   222 ItkProcWBodyLt(L2(16_2))
   313 16_00000000000000c6                   198 ItkProcSignatureLt([4,4])
   309 16_00000000000001a4                   420 ItkFormalsListLt(L5(16_5),[4,4])
   304 16_000000000000014e                   334 ItkVALUEFormalLt([4,6])
   300 16_00000000000001ac                   428 ItkVALUEFormalIdListLt(L2(16_2),[4,6])
   295 16_00000000000001b3                   435 ItkVALUEFormalIdListElem(Id3"F1",[4,6])
   290 16_00000000000001b3                   435 ItkVALUEFormalIdListElem(Id4"F2",[4,11])
   285 16_00000000000001af                   431 ItkVALUEFormalIdListRt(L2(16_2),[4,6])
   280 16_0000000000000152                   338 ItkVALUEFormalType([4,14])
   276 16_00000000000000b8                   184 ItkReservedId(Rid"INTEGER",[4,16])
   270 16_0000000000000155                   341 ItkVALUEFormalVal([5,4])
   266 16_00000000000000b1                   177 ItkFormalExprAbsent([5,4])
   262 16_0000000000000151                   337 ItkVALUEFormalRt([4,6])
   258 16_00000000000001a8                   424 ItkFormalsListSep(L2(16_2),[5,4])
   253 16_000000000000014e                   334 ItkVALUEFormalLt([5,12])
   249 16_00000000000001ac                   428 ItkVALUEFormalIdListLt(L1(16_1),[5,12])
   244 16_00000000000001b3                   435 ItkVALUEFormalIdListElem(Id5"F3",[5,12])
   239 16_00000000000001af                   431 ItkVALUEFormalIdListRt(L1(16_1),[5,12])
   234 16_0000000000000152                   338 ItkVALUEFormalType([5,15])
   230 16_00000000000000b8                   184 ItkReservedId(Rid"CHAR",[5,17])
   224 16_0000000000000155                   341 ItkVALUEFormalVal([6,4])
   220 16_00000000000000b1                   177 ItkFormalExprAbsent([6,4])
   216 16_0000000000000151                   337 ItkVALUEFormalRt([5,6])
   212 16_00000000000001a8                   424 ItkFormalsListSep(L3(16_3),[6,4])
   207 16_0000000000000162                   354 ItkROFormalLt([6,15])
   203 16_00000000000001bc                   444 ItkROFormalIdListLt(L1(16_1),[6,15])
   198 16_00000000000001c3                   451 ItkROFormalIdListElem(Id6"F4",[6,15])
   193 16_00000000000001bf                   447 ItkROFormalIdListRt(L1(16_1),[6,15])
   188 16_0000000000000166                   358 ItkROFormalType([6,18])
   184 16_00000000000000b8                   184 ItkReservedId(Rid"BOOLEAN",[6,20])
   178 16_0000000000000169                   361 ItkROFormalVal([7,4])
   174 16_00000000000000b1                   177 ItkFormalExprAbsent([7,4])
   170 16_0000000000000165                   357 ItkROFormalRt([6,6])
   166 16_00000000000001a8                   424 ItkFormalsListSep(L4(16_4),[7,4])
   161 16_0000000000000158                   344 ItkVARFormalLt([7,10])
   157 16_00000000000001b4                   436 ItkVARFormalIdListLt(L1(16_1),[7,10])
   152 16_00000000000001bb                   443 ItkVARFormalIdListElem(Id7"F5",[7,10])
   147 16_00000000000001b7                   439 ItkVARFormalIdListRt(L1(16_1),[7,10])
   142 16_000000000000015c                   348 ItkVARFormalType([7,13])
   138 16_00000000000000b8                   184 ItkReservedId(Rid"WIDECHAR",[7,15])
   132 16_000000000000015f                   351 ItkVARFormalVal([8,4])
   128 16_00000000000000b1                   177 ItkFormalExprAbsent([8,4])
   124 16_000000000000015b                   347 ItkVARFormalRt([7,6])
   120 16_00000000000001a8                   424 ItkFormalsListSep(L5(16_5),[8,4])
   115 16_00000000000001ac                   428 ItkVALUEFormalIdListLt(L1(16_1),[8,6])
   110 16_00000000000001b3                   435 ItkVALUEFormalIdListElem(Id8"F6",[8,6])
   105 16_00000000000001af                   431 ItkVALUEFormalIdListRt(L1(16_1),[8,6])
   100 16_00000000000000b0                   176 ItkFormalTypeAbsent([9,4])
    96 16_00000000000000b1                   177 ItkFormalExprAbsent([9,4])
    92 16_00000000000001a7                   423 ItkFormalsListRt(L5(16_5),[4,4])
    87 16_00000000000000b4                   180 ItkResultTypeAbsent([10,1])
    83 16_00000000000000b2                   178 ItkRaisesSetAbsent([10,1])
    79 16_00000000000000c9                   201 ItkProcSignatureRt([4,4])
    75 16_00000000000000e2                   226 ItkProcBodyLt([10,1])
    71 16_00000000000000ea                   234 ItkBlockLt(L4(16_4),[10,3])
    66 16_00000000000001c7                   455 ItkBlockDeclListRt(L0(16_0),[10,3])
    61 16_00000000000001c4                   452 ItkBlockDeclListLt(L0(16_0),[10,3])
    56 16_00000000000000ee                   238 ItkBlockBEGIN(L4(16_4),[10,3])
    51 16_00000000000001cf                   463 ItkStmtListRt(L0(16_0),[11,3])
    46 16_00000000000001cc                   460 ItkStmtListLt(L0(16_0),[11,3])
    41 16_00000000000000ed                   237 ItkBlockRt(L4(16_4),[10,3])
    36 16_00000000000000e5                   229 ItkProcBodyRt([10,1])
    32 16_00000000000000e1                   225 ItkProcWBodyRt(L2(16_2))
    29 16_00000000000001c7                   455 ItkBlockDeclListRt(L1(16_1),[3,2])
    24 16_00000000000000ee                   238 ItkBlockBEGIN(L1(16_1),[13,2])
    19 16_00000000000001cf                   463 ItkStmtListRt(L0(16_0),[14,2])
    14 16_00000000000001cc                   460 ItkStmtListLt(L0(16_0),[14,2])
     9 16_00000000000000ed                   237 ItkBlockRt(L1(16_1),[3,2])
     4 16_00000000000000a4                   164 ItkRightEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
